### PR TITLE
Test changes

### PR DESCRIPTION
**Pull request by anoopuchil**
Tested a  few style changes

### CODE REVIEW

File: src/app/ability-scores/ability-scores.component.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<div class="card">
+<div class="card" style="width: 50%;">
```
Comment (agrant216) [inline src/app/ability-scores/ability-scores.component.html:1]: Let's kepp styles to scss files

File: src/app/app.component.html [hunk 2]
```diff
@@ -1,5 +1,9 @@
 <general-info></general-info>
+<br><br>
```
Comment (agrant216) [inline src/app/app.component.html:2]: This should probably be accomplished with paddings and margins

File: src/app/general-info/general-info.component.html [hunk 3]
```diff
@@ -1,15 +1,37 @@
...
+    <input type="text" name="CharacterName" class="form-text">
+
+  </div>
+  
+  <div class="input-group">
+      <div class="input-group-prepend">
+        <label class="input-group-text" for="inputGroupSelectAlign">Alignment</label>
+      </div>
+      <select class="custom-select" id="Alignment01">
+        <option selected>Choose...</option>
```
Comment (agrant216) [inline src/app/general-info/general-info.component.html:16]: Can be converted to *ngFor with static list

**Review by agrant216 — APPROVED** [review]

File: src/app/ability-scores/ability-scores.component.html (hunk 1 above)
Comment (anoopuchil) [reply src/app/ability-scores/ability-scores.component.html:1]: Done

File: src/app/app.component.html (hunk 2 above)
Comment (anoopuchil) [reply src/app/app.component.html:2]: Done

**Pull request merged**